### DISCUSSION TITLE
`symbols-outline.nvim` not working for `qmd` file in Mac OS, but works fine with `md` file with the same contents.

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-nvim#111] [Q&A]
I have the same lazyvim based nvim config,[ oatVim4](https://github.com/hopezh/oatVim4/tree/symbols_outline_works), in which I use [symbols-outline.nvim](https://github.com/simrat39/symbols-outline.nvim) to get all the markdown headers in a `qmd` file.

Issue 1: The same nvim config works in Windows 11, but not in Mac OS (shown below).
<img width="1447" alt="Screenshot 2024-02-02 at 22 43 48" src="https://github.com/quarto-dev/quarto-nvim/assets/23435810/ff09f150-6a74-49ce-a73a-a5bda5cdff31">

Issue 2: Once I change the `test.qmd` file to `test.md`, `symbols-outline` works fine in Mac OS, as shown below. 
<img width="1447" alt="Screenshot 2024-02-02 at 22 44 01" src="https://github.com/quarto-dev/quarto-nvim/assets/23435810/5483fb14-a279-4251-b266-81dc9f287d9e">

So, may I ask how to make `qmd` file recognize by `symbols-outline` as just like a markdown file? 

Here are my [quarto.lua](https://github.com/hopezh/oatVim4/blob/symbols_outline_works/lua/plugins/quarto.lua) and [symbols-outline.lua](https://github.com/hopezh/oatVim4/blob/symbols_outline_works/lua/plugins/symbols_outline.lua) config files.

**Comment by jmbuhr** [comment]
is the marksman language server running in your quarto file (see https://github.com/hopezh/oatVim4/blob/1e17b3c1d6276b5335815f6ffe47a8f7ad2fd2c1/lua/plugins/quarto.lua#L211-L215)? You can verify with `:LspInfo`.

**Reply by hopezh** [reply]
Yes, I think the marksman lsp is running with the config as advised, as shown in the images below. 

<img width="1442" alt="Screenshot 2024-02-03 at 10 21 13" src="https://github.com/quarto-dev/quarto-nvim/assets/23435810/04daa10f-d723-4623-8aea-f52d231458b6">
<img width="657" alt="Screenshot 2024-02-03 at 10 22 07" src="https://github.com/quarto-dev/quarto-nvim/assets/23435810/d004c2aa-e53d-478d-8351-0eef2210cb91">

**Reply by jmbuhr** [reply]
hm, looks correct. Maybe iOS has a different version of marksman that does something differently? I would check that and then also clone the symbols-outline plugin to add a bunch of print statements to check if it is starting.

Otherwise, time to ditch the walled garden of apple and install Linux ;)
